### DISCUSSION TITLE
cache artificat is not uploaded to remote cache

### DISCUSSION THREAD

**Discussion by marwenguesmii** [vercel/turborepo#3242] [Help] [answered]
I enabled remote cache on my repo and when I run turbo build on my app-web. I can't find my cache artificat of the app-web in the remote cache but I do find all the dependency build caches. Meaning I am sure that the remote cache works. But why only the build of the app-web is not remote cached ?
I am using this to check the cache existance `https://api.vercel.com/v8/artifacts/<hash>?teamId=<hash>`with a Bearer Token
and the command turbo is running is `next build`

**Comment by anthonyshew** [comment, marked as answer]
Closing for staleness but this may have been a misconfiguration in `turbo.json`. If I were to guess, I would bet on the `outputs` key: https://turbo.build/repo/docs/reference/configuration#outputs

If you believe there is a bug in `turbo`, please open an issue.
